pkg/compare/parsing.go

```diff
@@ -101,24 +164,45 @@ func (r *Reference) getMissingCRs(matchedTemplates map[string]bool) (map[string]
 	return crs, count
 }
 
-var defaultFieldsToOmit = [][]string{{"metadata", "uid"},
-	{"metadata", "resourceVersion"},
-	{"metadata", "generation"},
-	{"metadata", "generateName"},
-	{"metadata", "creationTimestamp"},
-	{"metadata", "finalizers"},
-	{"kubectl.kubernetes.io/last-applied-configuration"},
-	{"metadata", "annotations", "kubectl.kubernetes.io/last-applied-configuration"},
-	{"status"},
+const builtInPathsKey = "cluster-compare-built-in"
+
+var builtInPaths = []*Path{
+	{PathToKey: "metadata.resourceVersion"},
+	{PathToKey: "metadata.generation"},
+	{PathToKey: "metadata.uid"},
+	{PathToKey: "metadata.generateName"},
+	{PathToKey: "metadata.creationTimestamp"},
+	{PathToKey: "metadata.finalizers"},
+	{PathToKey: `"kubectl.kubernetes.io/last-applied-configuration"`},
+	{PathToKey: `metadata.annotations."kubectl.kubernetes.io/last-applied-configuration"`},
+	{PathToKey: "status"},
+}
+
+type Path struct {
```

Comment:
Can you change the name to be more clear, and self explanatory to the type of the path (so its not confused with os path for example)